Marlin/Marlin.h

```diff
@@ -110,6 +110,8 @@ FORCE_INLINE void serialprintPGM(const char *str) {
 
 void get_command();
 void process_commands();
+uint16_t gcode_M238(uint8_t power = 0);
+void gcode_M241(long num_milliseconds = NULL);
```

Comment:
I believe those should only called within Marlin_main.cpp though. @jminardi what are your thoughts?